src/services/http.ts

```diff
@@ -1,10 +1,12 @@
-import { Logger } from './logger';
+import { Logger } from "./logger";
 
-import type { ApiConfig } from '../types';
+import type { ApiConfig } from "../types";
 export class HTTP {
   logger: Logger;
   apiConfig: ApiConfig;
 
+  static $inject = ["logger", "apiConfig"];
+
   constructor(apiConfig: ApiConfig) {
     this.apiConfig = apiConfig;
     this.logger = new Logger();
```

Comment:
you shouldn't create a logger instance by yourself, because it's already injected with the ioc container